packages/core/src/store/mad-config/hooks.ts

```diff
@@ -1,9 +1,21 @@
 import { getPremadeSettings } from "../../utils/getPremadeSettings";
 import { useMadConfig } from "./mad-config";
+import { getRedirectUriFromAuthConfig } from "../../utils/getRedirectUriFromAuthConfig";
 
-export const useAuthConfig = () => {
-    const config = useMadConfig();
-    return config.authentication;
+export type AuthConfig = {
+    clientId: string
+    redirectUri: string
+    scopes?: string[] | undefined
+}
```

Comment:
Should this type be used by `LoginButtonProps`?

Reply:
And in `MadConfig`?

Reply:
Madconfig looks slightly different, so I wouldn't add it there, but maybe loginbuttonprops

Reply:
AuthConfig now inherits some types from the mad-config. redirectUri is defined explicitly to avoid inheriting the JSDoc. LoginButtonProps is a part of `mad-auth` so it can't inherit types from this type